components/new-contribution-flow/index.js

```diff
@@ -67,9 +69,13 @@ class ContributionFlow extends React.Component {
     super(props);
     this.mainContainerRef = React.createRef();
     this.state = {
-      stepDetails: null,
       stepPayment: null,
       stepSummary: null,
+      stepDetails: {
```

Comment:
May or may not be some merge conflicts here - I also worked in this file but it looks like not any of the same lines, so hopefully that'll be ok 🤞